src/epgsql_binary.erl

```diff
@@ -313,4 +315,8 @@ default_oids() ->
      {timestamptz, 1184, 1185},
      {timetz, 1266, 1270},
      {uuid, 2950, 2951},
-     {varchar, 1043, 1015}].
+     {varchar, 1043, 1015},
+     {tsrange, 3908, 3909},
+     {tstzrange, 3910, 3911},
+     {daterange, 3912, 3913}
```

Comment:
May you, please, keep `default_oids()` and `default_codecs()` sorted? It's not required, but would help readability